src/metabase/query_processor/middleware/process_userland_query.clj

```diff
@@ -64,12 +64,12 @@
                                                              (catch Throwable e
                                                                (log/error e (trs "Error saving query execution info"))))))))
 
-(defn- save-successful-query-execution! [cache-details is_sandboxed? query-execution result-rows]
+(defn- save-successful-query-execution! [cache-details is-sandboxed? query-execution result-rows]
```

Comment:
We need a Kondo linter to keep people from using snake_case parameter names

(except maybe in `defendpoint` because that's how they come in)